.github/workflows/test-lint-github-worklows.yaml

```diff
@@ -0,0 +1,23 @@
+name: Lint test
+
+on:
+  workflow_dispatch:
```

Comment:
Do we need this realistically?

Reply:
Unlikely, but I don't think it can cause any harm keeping it